examples/hello-start_proc/input.ts

```diff
@@ -0,0 +1,7 @@
+export function hello() {
```

Comment:
Added a new example to demonstrate `start_proc` mode with multiple function outputs.